libraries/Microsoft.Bot.Builder.Dialogs/DialogManager.cs

```diff
@@ -116,13 +116,24 @@ public Dialog RootDialog
         /// </value>
         public int? ExpireAfter { get; set; }
```

Comment:
What is the purpose of this property? It doesn't seem to be used anywhere

Reply:
I think we could mark it as obselete, I don't know what it exists for either.